src/content/advanced/cost-optimization/workload-clusters-scale-down/cronjob-scale-down-silence.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: silences-customer-create-{CLUSTER-ID}
+  namespace: {YOUR_ORGANIZATION}
+  labels:
+    app: silence-customer
+spec:
+  schedule: "47 16 * * 5" #Timezone - UTC by default - Fridays, 16:47 UTC
+  successfulJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: silences-customer
+        spec:
+          serviceAccountName: silences-customer-rbac
+          containers:
+          - name: silences-customer-create-{CLUSTER-ID}
+            image: quay.io/giantswarm/k8s-initiator
+            imagePullPolicy: IfNotPresent
+            command:
+            - /bin/sh
+            - -c
+            - |
+              cat <<EOF | kubectl apply -f -
+              apiVersion: monitoring.giantswarm.io/v1alpha1
+              kind: Silence
+              metadata:
+                name: {INSTALLATION}-{CLUSTER-ID}-silence-cluster
+                annotations:
+                  'giantswarm.io/keep': 'true'
+              spec:
+                targetTags:
+                - name: installation
+                  value: {YOUR_INSTALLATION_NAME}
+                matchers:
+                - name: cluster_id
+                  value: {CLUSTER_ID}
+                  isRegex: false
+              EOF
+
+              # patch annotations of min and max values for autoscaling to 0
```

Comment:
Maybe we should say `YOUR_ORGANIZATION_NAMESPACE` rather than `YOUR_ORGANIZATION`? Also, we need the nodepool ID rather than the CLUSTER_ID